node/src/accountant/payment_adjuster/logging_and_diagnostics/log_functions.rs

```diff
@@ -0,0 +1,228 @@
+                Ord::cmp(&account_b.balance_wei, &account_a.balance_wei)
+            })
+            .map(|account| {
+                format!(
+                    "{} {}\n{:^length$} {}",
+                    account.wallet,
+                    original_account_balances_mapped
+                        .get(&account.wallet)
+                        .expectv("initial balance")
+                        .separate_with_commas(),
+                    BLANK_SPACE,
+                    account.balance_wei.separate_with_commas(),
+                    length = WALLET_ADDRESS_LENGTH
+                )
+            })
+            .join("\n")
+    }
+    fn format_summary_for_excluded_accounts(excluded: &[(&Wallet, u128)]) -> String {
+        let title = once(format!(
+            "\n{:<length$} Original\n",
+            "Ruled Out Accounts",
+            length = WALLET_ADDRESS_LENGTH
+        ));
+        let list = excluded
+            .iter()
+            .sorted_by(|(_, balance_account_a), (_, balance_account_b)| {
+                Ord::cmp(&balance_account_b, &balance_account_a)
+            })
+            .map(|(wallet, original_balance)| {
+                format!("{} {}", wallet, original_balance.separate_with_commas())
+            });
+        title.chain(list).join("\n")
+    }
+
+    let adjusted_accounts_wallets: Vec<&Wallet> = adjusted_accounts
+        .iter()
+        .map(|account| &account.wallet)
+        .collect();
+    let excluded: Vec<(&Wallet, u128)> = original_account_balances_mapped.iter().fold(
+        vec![],
+        |mut acc, (wallet, original_balance)| {
+            if !adjusted_accounts_wallets.contains(&wallet) {
+                acc.push((wallet, *original_balance));
+            }
+            acc
+        },
+    );
+    let adjusted_accounts_summary =
+        format_summary_for_included_accounts(&original_account_balances_mapped, adjusted_accounts);
+    let excluded_accounts_summary_opt = excluded
+        .is_empty()
+        .not()
+        .then(|| format_summary_for_excluded_accounts(&excluded));
+    vec![
+        Some(adjusted_accounts_summary),
+        excluded_accounts_summary_opt,
+    ]
+    .into_iter()
+    .flatten()
+    .join("\n")
+}
+
+pub fn accounts_before_and_after_debug(
+    original_account_balances_mapped: HashMap<Wallet, u128>,
+    adjusted_accounts: &[PayableAccount],
+) -> String {
+    format!(
+        "\n\
+            {:<length$} {}\n\
+            \n\
+            {:<length$} {}\n\
+            {:<length$} {}\n\
+            \n\
+            {}",
+        "Payable Account",
+        "Balance Wei",
+        BLANK_SPACE,
+        "Original",
+        BLANK_SPACE,
+        "Adjusted",
+        format_brief_adjustment_summary(original_account_balances_mapped, adjusted_accounts),
+        length = WALLET_ADDRESS_LENGTH
+    )
+}
+
+pub fn info_log_for_disqualified_account(
+    logger: &Logger,
+    account: &DisqualificationSuspectedAccount,
+) {
+    info!(
```

Comment:
It seems that it could be overwhelming to user to see so many info notifications for each account. Ideally, it should be a summary along the lines, "X accounts haven't been paid yet, due to low MASQ balance."